examples/function-pointers/function_pointers.bal

```diff
@@ -8,7 +8,8 @@ function test(int x, string s) returns float {
     if (y is int) {
         f = x * 1.0 * y;
     } else {
-        // Runtime value is cast to correct type since Ballerina runtime can infer the correct type to error.
+        // The runtime value is cast to the correct type since Ballerina
+        // runtime can infer the correct type to error.
```

Comment:
Shall we just say something that indicates that the type of `y` within the `else` block would be `error`, since the `int` case is already handled. Casting is too much info right?

Reply:
Changed with https://github.com/ballerina-platform/ballerina-lang/pull/14911/commits/d3c8c5a9532002ca935759535f1443f36accc8b6